frontend/src/modules/S3_Datasets/views/DatasetView.js

```diff
@@ -266,13 +266,14 @@ const DatasetView = () => {
 
             <Grid item>
               <Box sx={{ m: -1 }}>
+                <UpVoteButton
+                  disabled={!isAdmin}
+                  upVoted={isUpVoted}
+                  onClick={() => upVoteDataset(dataset.datasetUri)}
+                  upVotes={upVotes}
+                />
```

Comment:
does `UpVote` take in a `disabled` prop / does this really do anything?

Reply:
It just shows the number of votes, it allows users to read the upVotes but not to upvote

Reply:
Pretty sure I was a user who had share access to the dataset and was still able to execute upVote - only being stopped on the server side

Reply:
Fixed, thank you!
![image](https://github.com/user-attachments/assets/8d6caeae-93d2-4780-8ab5-32b08715cc0d)